sql/createtables.sql

```diff
@@ -0,0 +1,124 @@
+USE [tree-db]
```

Comment:
Maybe better to leave this out -- rather than hard-coding the database name we can delete this line and that will allow for multiple databases on the server (e.g. a test database)

Reply:
makes sense